daffodil-core/src/main/java/org/apache/daffodil/api/validation/Validators.java

```diff
@@ -15,49 +15,61 @@
  * limitations under the License.
  */
 
-package org.apache.daffodil.lib.validation
+package org.apache.daffodil.api.validation;
 
-import org.apache.daffodil.lib.api.ValidatorFactory
-import org.apache.daffodil.lib.util.SimpleNamedServiceLoader
+import org.apache.daffodil.util.SimpleNamedServiceLoader;
+
+import java.util.Optional;
 
 /**
- * Access SPI registered [[org.apache.daffodil.lib.api.ValidatorFactory]] instances.
- *
+ * Access SPI registered [[org.apache.daffodil.api.validation.ValidatorFactory]] instances.
+ * <p>
  * Registered instances provide a unique name for lookup.
  */
-object Validators {
-  private lazy val impls: Map[String, ValidatorFactory] = {
-    SimpleNamedServiceLoader.loadClass[ValidatorFactory](classOf[ValidatorFactory])
+public class Validators {
+  private static final Validators INSTANCE = new Validators();
+  private final java.util.Map<String, ValidatorFactory> impls =
+      SimpleNamedServiceLoader.loadClass(ValidatorFactory.class);
+
+  private Validators() {
+  } // private constructor
+
+  public static Validators getInstance() {
+    return INSTANCE;
   }
 
   /**
    * Get the factory by name or throw
+   *
    * @param name registered name of the validator factory
+   * @return {@link org.apache.daffodil.api.validation.ValidatorFactory} the factory instance
    * @throws ValidatorNotRegisteredException when factory is not found in the registered services
-   * @return [[org.apache.daffodil.lib.api.ValidatorFactory]] the factory instance
    */
-  @throws(classOf[ValidatorNotRegisteredException])
-  def get(name: String): ValidatorFactory =
-    impls.getOrElse(name, throw ValidatorNotRegisteredException(name))
+  public ValidatorFactory get(String name) throws ValidatorNotRegisteredException {
```

Comment:
Can we make all these functions static so no one every needs to do `Validators.getInstance` first? They can just do things like `Validators.get(..)` and the first thing that will do is call getInstance? Should simplify uses.